compiler/src/iree/compiler/Codegen/LLVMGPU/Utils/LLVMGPULayoutAnalysisAndDistribution.cpp

```diff
@@ -785,86 +796,87 @@ static void distributeReductionBroadcastTranspose(
   }
 
   bodyType loopBody = [&](std::array<int, DimType::NumDims> &state) {
+    Value vector = simdToSimtMap.at(source);
+    VectorType vectorType = vector.getType().cast<VectorType>();
+    Type elementType = vectorType.getElementType();
+    int bitWidth = elementType.getIntOrFloatBitWidth();
+    bool isFP32 = bitWidth == 32;
```

Comment:
the name seems misleading, either you want to check for f32 only and you should do elementType.IsFloat32Ty() or if you want to allow i32 type also you should pick a better name (`is32BitsElement`?)

Reply:
Good catch! Yes I want to check for f32 only so I will change this to use elementType.isF32().